src/main.ts

```diff
@@ -64,19 +72,38 @@ export class Minifier {
     }
   }
 
+  // For testing purposes ONLY. Do not want to actually create/write
+  // the new TS files, only want to return the string.
+  renameProgramForTesting(node: ts.Node): string {
```

Comment:
This can be useful for more than testing, example another tool that does its own TS compilation and just wants to pass-in a Node and expects a string back.

How about 'renameProgramFromNode', also add a comment that this expects the typechecker to have been set using setTypeChecker.